<?xml version="1.0"?>
<launch>
  <!-- Tags of planner node -->
  <node
    pkg="the_explorer_robot"
    type="planner"
    name="planner"
  />
  <!-- Tags of plannerTest -->
  <test 
    pkg="the_explorer_robot" 
    test-name="plannerTest" 
    type="plannerTest" 
  />
</launch>

